src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd

```diff
@@ -134,6 +134,7 @@
     <xsd:attribute name="decoration-priority" type="xsd:integer" />
     <xsd:attribute name="autowire" type="boolean" />
     <xsd:attribute name="autoconfigure" type="boolean" />
+    <xsd:attribute name="error" type="xsd:string" />
```

Comment:
Dumping error is good for us, but that also mean it can be defined by users with this new attribute (preferred to keep things simple with an attribute btw). We open this new possibility. I'm not sure it's good 🤔 That makes it a feature too? That means it can't really go on 4.4 and that we can't fix the linked issue like this? I need some input on all of this, thanks.